fdbserver/RestoreUtil.h

```diff
@@ -34,6 +34,9 @@
 #include <cstdint>
 #include <cstdarg>
 
+// #define SevFRMutationInfo  SevNoInfo
+#define SevFRMutationInfo SevInfo
```

Comment:
Did you want to commit this to master at `SevInfo` severity or at the lower severity?